utils/createUrlParams.ts

```diff
@@ -3,13 +3,21 @@
  * @param params
  */
 export function createUrlParams(params: {
-  [x: string]: string | string[] | number | boolean;
+  [x: string]: string | number | boolean;
```

Comment:
Would this work when we also have `labels ` to pass as a parameter?

Reply:
My expectation was that you would pass multiple labels as e.g. `rl=12&rl=42` - multiple instances of the parameter name, which is how Postman and forms seem to construct such collection parameters usually. That's why I made the labels and users params two letter names.

Reply:
This is a valid point. But my concern is not about the input type, but of line 10 here.

I'm not sure if it would work, but we probably could remove this method altogether since the format the APIs requesting looks like the same as our URL. 

Let's take a look at it when implementing labels.

Reply:
Sounds good!